<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1352</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AddTechnique">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Technique</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Tactics">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>281</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
   </widget>
   <widget class="QComboBox" name="Techniques">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>281</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="TacticsBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="plainText">
     <string>Tactics</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="TechniquesBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Techniques</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="DescriptionBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>521</width>
      <height>461</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="PotentialMatches">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>470</y>
      <width>521</width>
      <height>75</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Potential Matches: #</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pick">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>490</y>
      <width>102</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Pick</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteTechnique">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>470</y>
      <width>241</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Technique</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="SoftwareBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Software</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Software">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>281</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPushButton" name="AddSoftware">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Software</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AddedTechniquesText">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Added Techniques</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteSoftware">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>470</y>
      <width>241</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Software</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AddedSoftwareText">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>0</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Added Software</string>
    </property>
   </widget>
   <widget class="QListWidget" name="AddedTechniquesBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>60</y>
      <width>241</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="AddedSoftwareBox">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>60</y>
      <width>241</width>
      <height>401</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>35</red>
          <green>39</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>35</red>
          <green>39</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>147</green>
          <blue>147</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearAll">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>510</y>
      <width>491</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
   <zorder>PotentialMatches</zorder>
   <zorder>DeleteTechnique</zorder>
   <zorder>DeleteSoftware</zorder>
   <zorder>AddedSoftwareText</zorder>
   <zorder>AddedSoftwareBox</zorder>
   <zorder>ClearAll</zorder>
   <zorder>TacticsBox</zorder>
   <zorder>TechniquesBox</zorder>
   <zorder>SoftwareBox</zorder>
   <zorder>Tactics</zorder>
   <zorder>Techniques</zorder>
   <zorder>Software</zorder>
   <zorder>AddTechnique</zorder>
   <zorder>AddSoftware</zorder>
   <zorder>DescriptionBox</zorder>
   <zorder>Pick</zorder>
   <zorder>AddedTechniquesText</zorder>
   <zorder>AddedTechniquesBox</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1352</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUpdate"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
